### ISSUE TITLE
Error compiling go program using gtk

### ISSUE THREAD

**Issue by nicolasbrigodiot** [tpoechtrager/osxcross#61]
Hi,

I am trying to compile a go program using the osxcross toolchain:

```
env OSXCROSS_NO_INCLUDE_PATH_WARNINGS=1 MACOSX_DEPLOYMENT_TARGET=10.6 CC=o64-clang CXX=o64-clang++ GOOS=darwin GOARCH=amd64 CGO_ENABLED=1 go build -v mywork/myprogram
```

I get the following (linking?) errors:

```
...
os/exec
# github.com/mattn/go-gtk/pango
ld: library not found for -lpango-1.0
clang: error: linker command failed with exit code 1 (use -v to see invocation)
io/ioutil
github.com/mattn/go-gtk/glib
math/big
encoding/hex
compress/flate
compress/gzip
log
mime
crypto/elliptic
encoding/asn1
crypto/rand
# github.com/mattn/go-gtk/glib
ld: library not found for -lgobject-2.0
clang: error: linker command failed with exit code 1 (use -v to see invocation)
crypto/dsa
...
```

I must say I am a newbie to the Linux world and compilers and linkers in general and I don't understand everything I've been doing so far but I know I can compile successfully using the default go compiler (for a native linux target, I'm running Ubuntu) so I understand the problem here lies within the o64-lang compiler - my understanding is that it is unable to find the pango and gobject libraries?

Those libraries are used by the go-gtk library which I use myself, so I have asked the author of the go-gtk library if they knew what to do about it (https://github.com/mattn/go-gtk/issues/255) and apparently their suggestion is to compile the libraries myself using the osxcross compiler but honestly I am not sure what that means.

Any help (with detailed commands) would be greatly appreciated as I have no idea what to do here.

Thanks!

**Comment by thomasf** [comment]
Yes, that is correct, the linker `ld` needs the osx versions of those libraries to be able to link your program.

You should probably start by reading up on how a normal C/C++ build works to get an overview of this..
Then you should probably (I don't know) go here http://www.gtk.org/download/macos.php .

I suggest you set this up on a native osx desktop machine before trying
